reference/orders.v3.yml

```diff
@@ -2337,7 +2334,7 @@ definitions:
           - ORDER
       item_id:
         type: integer
-        description: Reference to the item
+        description: order_product.id corresponding to the item_types of PRODUCT, GIFT_WRAPPING. order_address.id corresponding to the item_types of SHIPPING, HANDLING. order.id corresponding to the item_type of ORDER.
```

Comment:
This is definitely an improvement
You have more freedom here if you go with the block syntax using `|-`
see https://github.com/WillemHoman/api-specs/blob/d1ce2955924ee71e63604e8ca5e193fe8c4ea743/reference/orders.v3.yml#L73

You could then probably even insert a table for this like

https://github.com/WillemHoman/api-specs/blob/d1ce2955924ee71e63604e8ca5e193fe8c4ea743/reference/orders.v3.yml#L27